### DISCUSSION TITLE
Consommation horaires pour ENGIE

### DISCUSSION THREAD

**Discussion by jamesbondvonneuman** [JC144/EDF_Simulateur_Prix#19] [Ideas]
Pour les titulaires d'un contrat chez ENGIE, il n'existe pas, dans l'onglet "Consommation"de l'espace client, de fichier .csv relatif aux consommations horaires. Bien dommage. Comment peut-on faire pour y remédier ? (j'ai contacté ENGIE et j'attends leur réponse). Bien cordialement

**Comment by JC144** [comment]
Merci de les avoir contacté. N'hésitez pas à partager leur retour quand vous l'aurez.

Est-ce que vous avez essayé en passant par Enedis ? Je ne suis pas sûr du fonctionnement exact, mais je me demande si les données ne transitent pas par Enedis avant d'arriver chez le fournisseur d'énergie.
Chez EDF, on a un export depuis le site d'EDF et un autre depuis le site d'Enedis par exemple, mais les deux entités étaient proches historiquement.

**Comment by jamesbondvonneuman** [comment]
Bonjour.
En effet, avec le sous-traitant ENEDIS, j'ai toutes les données  pour mon compte ENGIE.
Très cordialement.
